index.js

```diff
@@ -422,9 +422,9 @@ function buildJumpMap (segments) {
           )
           (else 
             ;; return callback destination and zero out $cb_dest 
-            get_global $cb_dest
+            (set_local $jump_map_switch (get_global $cb_dest))
```

Comment:
You need to declare the local in the above function next to `(local jump_dest i32)`

Reply:
fixed